diff --git NixNote2.pro NixNote2.pro
index 1bb6de5..1e50491 100644
--- NixNote2.pro
+++ NixNote2.pro
@@ -435,3 +435,11 @@ HEADERS  += nixnote.h \
 
 QMAKE_CXXFLAGS +=-g -O2 -fstack-protector --param=ssp-buffer-size=4 -Wformat -Werror=format-security
 QMAKE_LFLAGS += -Wl,-Bsymbolic-functions -Wl,-z,relro
+
+binary.path = /usr/bin/
+binary.files = nixnote2
+
+desktop.path = /usr/share/applications/
+desktop.files = nixnote2.desktop
+
+INSTALLS = binary desktop
